use-timescale/extensions/postgis.md

```diff
@@ -0,0 +1,160 @@
+---
+title: postgis PostgreSQL extension
+excerpt: Use the postgis extension with your Timescale service
+products: [cloud]
+keywords: [services, settings, extensions, postgis]
+tags: [extensions, postgis]
+---
+
+# The `postgis` extension
+
+The `postgis` PostgreSQL extension provides storing, indexing and querying
+geographic data. It helps in spatial data analysis, the study of patterns,
+anomalies, and theories within spatial or geographical data.
+
+For more information about these fuctions and the options available, see the
+[PostGIS documentation] [postgis-docs].
+
+## Use `postgis` extension with Timescale
+
+`postgis` is a PostgreSQL extension that enables you to conduct complex analyses
+of your geospatial time-series data. Timescale understands multitude of data
+challenges and helps you with when things happened and where they occurred.
```

Comment:
```suggestion
of your geospatial time-series data. Timescale understands that you have a multitude of data
challenges and helps you discover when things happened, and where they occurred.
```

Reply:
I actually think we might need a little understanding of the example here, because it sort of comes out of the blue in the procedure. Something like "This example uses a dataset that ..."?